tap_tiktok_ads/streams.py

```diff
@@ -252,7 +252,7 @@ def get_date_batches(self, stream_id, advertiser_id):
             Returns batches with start_date and end_date for the date_windowing using bookmark/start_date
         """
         if ('bookmarks' in self.state) and (stream_id in self.state['bookmarks'] and (str(advertiser_id) in self.state['bookmarks'][stream_id])):
-            start_date = parse(self.state['bookmarks'][stream_id][str(advertiser_id)]) + timedelta(days=1)
+            start_date = parse(self.state['bookmarks'][stream_id][str(advertiser_id)])
```

Comment:
I don't understand why they had time-delta of 1 day. it was added in this [commit](https://github.com/singer-io/tap-tiktok-ads/commit/4d198c1ba07033b0635ffe6382cabfc8eaacef01)

Reply:
@RushiT0122 and I discussed this issue, we could't find any reasoning to it.
but the adverse effect is that it creates a gap window which is not synced so we have reverted this timedelta.
Please find the related ticket [here](https://jira.talendforge.org/browse/TDL-21907)